src/lib/types/services.ts

```diff
@@ -89,9 +91,8 @@ export interface IUnleashServices {
     instanceStatsService: InstanceStatsService;
     favoritesService: FavoritesService;
     maintenanceService: MaintenanceService;
-    /** @deprecated prefer exportImportServiceV2, we're doing a gradual rollout */
-    exportImportService: ExportImportService;
-    exportImportServiceV2: WithTransactional<ExportImportService>;
+    exportService: IExportService;
+    importService: WithTransactional<ExportImportService>;
```

Comment:
Why not IExportService after you split them?

Reply:
I believe you mean, why not IImportService, right? Now, export doesn't have to be transactional and we can use the same and we only need import to be transactional

Reply:
Let me know if [6950fc1](https://github.com/Unleash/unleash/pull/5004/commits/6950fc1962255f15b0f246a1c481915f480f6e39) is what you're expecting ;)

Reply:
correct :)